framework/src/problems/FEProblemBase.C

```diff
@@ -5350,6 +5385,20 @@ FEProblemBase::needBoundaryMaterialOnSide(BoundaryID bnd_id, THREAD_ID tid)
   return _bnd_mat_side_cache[tid][bnd_id];
 }
 
+bool
+FEProblemBase::needBoundaryMaterialOnInterafce(BoundaryID bnd_id, THREAD_ID tid)
+{
+  if (_bnd_mat_interface_cache[tid].find(bnd_id) == _bnd_mat_interface_cache[tid].end())
+  {
+    _bnd_mat_interface_cache[tid][bnd_id] = false;
+
+    if (_interface_user_objects.hasActiveBoundaryObjects(bnd_id, tid))
```

Comment:
I want to see another call similar to what's in `FEProblemBase::needBoundaryMaterialOnSide`, something like:
```
if (_nl->needBoundaryMaterialOnInterface(bnd_id, tid))
```
and the above would check whether there are any active `_interface_kernel` objects